<launch>
  
  <!--
    Leds 1
  -->
  <node pkg="nao_behavior_tree" type="nao_leds" name="nao_leds1" output="screen">
    <rosparam command="load" file="$(find nao_behavior_tree)/launch/parameters.yaml"/>
    <param name="NAO_IP" value="192.168.1.19"/>
    <param name="NAO_PORT" value="9559"/>
  </node>
  
  <!--
    Leds 2
  -->
  <node pkg="nao_behavior_tree" type="nao_leds" name="nao_leds2" output="screen">
    <rosparam command="load" file="$(find nao_behavior_tree)/launch/parameters.yaml"/>
    <param name="NAO_IP" value="192.168.1.127"/>
    <param name="NAO_PORT" value="9559"/>
  </node>
  
  
  <!--
    Localization
  -->
  <node pkg="nao_behavior_tree" type="Localization" name="Localization" args="0" output="screen">
    <rosparam command="load" file="$(find nao_behavior_tree)/launch/webcam.yaml"/>
    
    <param name="estimatedPosition" value="true"/>
    <param name="theta_01" value="0"/>
    <param name="theta_02" value="3.14159265"/>
    <param name="NAO_IP1" value="192.168.1.19"/>
    <param name="NAO_PORT1" value="9559"/>
    <param name="NAO_IP2" value="192.168.1.127"/>
    <param name="NAO_PORT2" value="9559"/>
  </node>
  
  
  <!--
    Behavior Tree
  -->
  <node pkg="nao_behavior_tree" type="nao_behavior_tree" name="nao_behavior_tree1" output="screen">
    <param name="file" value="/home/olivier/ros_workspace/nao_behavior_tree/bt_localization.txt"/>
  </node>

</launch>
